test/coverage-system/coverageservice.test.ts

```diff
@@ -0,0 +1,60 @@
+import * as assert from "assert";
+import {OutputChannel, workspace} from "vscode";
+
+import {CoverageService} from "../../src/coverage-system/coverageservice";
+
+const mockOutputChannel = {appendLine: (x) => {}} as OutputChannel;
+const mockFileWatcher = {
+    dispose: () => {},
+    onDidChange: (fn) => {},
+    onDidCreate: (fn) => {},
+    onDidDelete: (fn) => {},
+};
```

Comment:
nice job imitating the other mocker style bits from the existing tests.
I was going to use sinon or jest mocks eventually but need to update the vscode test harness first before being able to bring in something like jest.